ldms/src/ldmsd/ldmsd_updtr.c

```diff
@@ -636,20 +655,24 @@ static void schedule_prdcr_updates(ldmsd_updtr_task_t task,
 			if (rc) {
 				/* If the error is EEXIST, the set is already in the set tree. */
 				if (rc == EEXIST) {
-					ldmsd_log(LDMSD_LERROR, "Prdcr '%s': "
-						"lookup failed synchronously. "
-						"The set '%s' already exists. "
-						"It is likely that there are more "
-						"than one producers pointing to "
-						"the set.\n",
-						prd_set->prdcr->obj.name,
-						prd_set->inst_name);
+					warn_once_prdcr_set_exists(prd_set);
```

Comment:
Again this is a serious error.

Reply:
The fact that it's serious doesn't mean we should flood the log with it. Logging it once per set on which it occurs.

Reply:
Jim has requested a presentation and discussion at the monday ldms user meeting.
I will put some ppt together.